src/code/fv-components/breed-button.js

```diff
@@ -0,0 +1,31 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+import t from '../utilities/translate';
+
+const BreedButtonView = React.createClass({
+  propTypes: {
+    onClick: PropTypes.func.isRequired,
+    disabled: PropTypes.bool
+  },
+
+  handleClick: function() {
+    this.props.onClick();
+  },
+
+  render: function() {
+    return (
+      <div className='geniblocks breed-button'>
+        <div className={classNames('clickable-button', { 'disabled': this.props.disabled})} onClick={this.handleClick}>
+          <div className="text unselectable">{ t("~FV_EGG_GAME.BREED_BUTTON").toUpperCase() }</div>
```

Comment:
I would leave off the call to `toUpperCase` here. The author of the string should be able to choose whether the string is capitalized or not without having the code override the author's choice. I'll comment on class names in the CSS file where they're used.